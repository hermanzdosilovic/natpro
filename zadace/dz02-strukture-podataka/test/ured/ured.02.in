100
7 49 23 8 30 22 44 28 23 9 40 15 42 42 37 3 27 29 40 12 3 19 9 7 10 33 49 28 16 35 47 26 12 17 10 33 29 49 29 21 17 22 43 36 35 45 28 41 44 7 1 3 8 44 18 40 24 46 30 3 22 16 49 24 1 3 27 8 28 33 48 31 35 13 15 14 13 36 25 19 15 44 29 1 17 45 5 4 1 48 38 23 5 32 2 16 16 37 38 44
20
sef 98 100
promjena 38 8
sef 98 98
promjena 10 31
sef 76 86
promjena 43 49
promjena 13 10
sef 95 99
sef 69 78
promjena 7 12
promjena 66 32
promjena 68 21
sef 13 36
sef 91 92
promjena 58 16
promjena 41 29
sef 7 31
promjena 96 19
promjena 24 39
promjena 6 26
