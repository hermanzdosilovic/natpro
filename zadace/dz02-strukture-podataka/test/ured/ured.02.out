37
37
1
2
13
3
23
3
